static/styles/right-sidebar.scss

```diff
@@ -30,6 +30,10 @@
     list-style-position: inside; /* Draw the bullets inside our box */
 }
 
+#group-pms {
+    max-width: 95%;
+}
```

Comment:
Why do we need the `max-width` here?

Reply:
This aligns the background-color with the `Users` on hover.
just like `#user_presences {max-width:95%; }` 

![image](https://user-images.githubusercontent.com/1678423/49437810-cffa4d80-f7bc-11e8-9f62-b05328ba06b6.png)
![image](https://user-images.githubusercontent.com/1678423/49437840-e3a5b400-f7bc-11e8-9388-06105650c9bb.png)